ext/simplexml/simplexml.c

```diff
@@ -2631,15 +2629,21 @@ PHP_FUNCTION(simplexml_import_dom)
 		if (nodep->type == XML_DOCUMENT_NODE || nodep->type == XML_HTML_DOCUMENT_NODE) {
 			nodep = xmlDocGetRootElement((xmlDocPtr) nodep);
 		}
+	} else {
```

Comment:
`if (!nodep)` early return?

Reply:
I haven't realized until now that it's a good opportunity to use an early return... but I like the idea!